Source/WebKit/Configurations/WebKit.xcconfig

```diff
@@ -234,6 +239,7 @@ WK_MIGRATED_HEADER_OUTPUT_DIR_cocoatouch = $(HEADER_OUTPUT_DIR:dir)PrivateHeader
 EXCLUDED_MACOS_PLUGIN_FILE_NAMES[sdk=iphone*] = SecItemShim.dylib WebProcessShim.dylib *.pdf Resources/mac/* com.apple.WebKit.NetworkProcess.sb com.apple.WebKit.GPUProcess.sb com.apple.WebKit.webpushd.mac.sb com.apple.WebProcess.sb;
 
 INSTALLHDRS_SCRIPT_PHASE = YES;
+INSTALLHDRS_COPY_PHASE = YES;
```

Comment:
There's a non-zero chance that this will impact production builds, but if testing shows it working fine, then LGTM. (cc @aproskuryakov)

Reply:
Should probably diff an installhdrs root to verify what changes.

Reply:
Diff'ed installhdrs roots and they look good.